private-templates-service/README.md

```diff
@@ -68,29 +69,36 @@ app.use(function (req, res, next) {
     "GET, POST, OPTIONS, DELETE"
   );
   next();
-});`
-
-**4. In the `adaptivecards-templates/private-templates-service/server/app.ts` file, replace ` "#{DB_CONNECTION_TOKEN}#"` with the real DB connection string. Make sure to NOT commit this file.**
-
-**5. Under the `adaptivecards-templates/private-templates-service/server`, run `npm run link-client`. Please note you may have to run  `sudo npm run link-client` if you are having errors with permissions.**
+});
+```
 
-**6. Inside of `adaptivecards-templates/private-templates-service/server` run `npm install`, run `tsc`, and run `npm run dev` to launch the client and server in your default browser.**
+5. `cd adaptivecards-templates/private-templates-service/server` and run `npm run init-app`. This installs and links dependencies.
 
-**You're good to go!**
+6. Run `npm run dev`. This command concurrently runs the client and server locally. Navigate to `localhost:3000` to see the site.
 
-# Running the backend
+## Running the latest changes without MongoDB
 
-To run this web application locally, you must have the following installed on your system:
+Prerequisites:
 
 -   Git
 -   [Node v12](https://nodejs.org/en/download/)
 
-**1. Clone the repository.**
+1. Clone the repository.
 
-**2. Switch to the appropriate branch.**
+2. Switch to the desired branch. The latest build is on `dev`.
+
+3. Set the below environment variables to the desired values. 
+
+```
+ACMS_APP_ID // Azre Active Directory App Registration Application (client) id 
+ACMS_REDIRECT_URI // http://localhost:3000
+```
+
+**4. Open `server/app.ts` and replace lines 39 - 64 with the following:**

```

Comment:
Can we also not use a line number here please? Could do something similar as mentioned above.